elo_loyalty_prediction.ipynb

```diff
@@ -181,7 +407,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.7.2"
+   "version": "3.7.1"
```

Comment:
Why this change?

Reply:
Whoops, that was not on purpose – I suppose it did it automatically since I only have v3.7.1 installed on this computer.

@boris-tschirschwitz Whoops, seems that I once again pretended to forget to add the requirements. ;-)

I have now upgraded to v3.7.2 here, added the missing requirements and completed the half-finished subordinate clause (in addition to making some other revisions of the explanatory texts).